getaplotdevelopment/evently-backend
# Create event with ticket category

#### Description of changes made
- Create events with ticket categories

#### Type of change

#### How Has This Been Tested?
- After cloning the [repo](https://github.com/getaplotdev/evently-backend.git)
- Check out this branch by `branch here`
- Run `yarn or npm install` to install the project dependencies
- Run 'action to be done`


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

#### Checklist


#### GitHub issue your changes fix

#### Screenshots
<img width="1013" alt="Screenshot 2020-03-16 at 12 45 01" src="https://user-images.githubusercontent.com/32167860/76743638-feb6f000-6783-11ea-9a41-ea110a010453.png">
